gapic/api/artman_bigtable.yaml

```diff
@@ -1,5 +1,6 @@
 common:
-  api_name: google-cloud-bigtable-v2
+  short_name: bigtable
+  version: v2
```

Comment:
Yeah I think you're right that it's verbose - `api_name` makes sense. Potentially the full name in artman should be `api_full_name` (not relevant to this config, but relevant to the overall naming).